contracts/ictt/README.md

```diff
@@ -0,0 +1,94 @@
+# Avalanche Interchain Token Transfer (ICTT)
```

Comment:
We should also move the teleporter readme and add a top level readme explaining the overall repo structure and types of contracts it contains but we can enter a separate issue for that